cypress/constants/common.ts

```diff
@@ -0,0 +1,7 @@
+export const DATA_SERVICES = 'Data Services'; // nav menu category option
+export const DATA_FEDERATION = 'Data Federation'; // nav menu sub category option
+export const STATUS_CARD_TITLE = 'Data Federation service';
+
+export const NO_WAIT = {
+  timeout: 0,
+};
```

Comment:
don't add this...

Reply:
why?

Reply:
wherever I have added, there is no point in refreshing or reloading. we can compute faster

Reply:
u don't have to add it... cypress will take care of it... this will add flakyness to tests...
default timeout does not mean cypress will wait for that time before moving forward...

Reply:
but I added it to not wait when it is wrong.

example when the location URL pathname is wrong, then also it is waiting and trying to check which does not yield anything as the location URL pathname is not going to change

Reply:
example  [here](https://github.com/debjyoti-pandit/mcg-ms-console/blob/f0224f026d85c92d9f94618a2b25716ce0988dfb/cypress/tests/status-card.spec.ts#L18) -

when we have already selected the HTML element containing the SVG of the status icon.
then why wait for 4 seconds when we don't see status icon not green.

Reply:
also I don't like the way we are checking .contains() and then inside older block again using .contains on same element, this is not right way to chain...

Reply:
> as I said above, timeout does mean cypress will wait for that time before moving forward... if it finds the element in 0.5 sec it will move forward as soon as that is complete... timeout !== sleep

but what if we don't find the element then it will wait for useless 4 seconds

Reply:
it is supposed to do so... that is reason of using cypress...

Reply:
instead of adding wait improve the code (tests)...

Reply:
> instead of adding wait improve the code (tests)...

ok but even if our code is great. then also if it is going to fail it will wait for 4 seconds. where we can avoid the waiting

Reply:
example: consider a button with the name deb.
if we check via contains then it is waiting for 4 seconds when it is always going to fail.
so as soon as it fail we should not retry

Reply:
if u already know it is going to fail... add `should` accordingly... that u r expecting it to fail... that is what I mean by improving code (tests)...